assets/css/frontend/sensei.scss

```diff
@@ -521,88 +556,6 @@ section.entry span.course-lesson-progress { margin-left: 10px; }
       width: 90%;
     }
   }
-  &.ui-tabs {
```

Comment:
What about people who are currently using these classes to hook into this UI?

Reply:
We should probably bump the version of the template where this is used. We can also use some of these class names to keep compatibility, but UI tabs also adds classes for things like hover and focus state, which I would avoid simulating.